python/tritonserver/__init__.py

```diff
@@ -23,3 +23,66 @@
 # OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+"""In process Python API for Triton Inference Server."""
+
+import tritonserver._c as _triton_bindings
+from tritonserver._api._datautils import DataType as DataType
+from tritonserver._api._datautils import MemoryAllocator as MemoryAllocator
+from tritonserver._api._datautils import MemoryBuffer as MemoryBuffer
+from tritonserver._api._datautils import MemoryType as MemoryType
+from tritonserver._api._datautils import NumpyAllocator as NumpyAllocator
```

Comment:
The [TorchAllocator](https://github.com/triton-inference-server/tutorials/blob/cca6b1998d40e2e0539dc03adc6cb9ec0ccb67e4/RayServe/server.py#L36-L50) would make a great example for the docs around allocators if it's not going to be shipped by default